sweepai/utils/utils.py

```diff
@@ -166,53 +216,8 @@ def count_length_without_whitespace(s: str):
     timeout=10,
 )
 class Chunking:
-
     def __enter__(self):
-        from tree_sitter import Language
-
-        logger.debug("Downloading tree-sitter parsers")
-
         LANGUAGE_NAMES = ["python", "java", "cpp", "go", "rust", "ruby", "php"]
```

Comment:
move this to a shared constant. __enter__ needs self.languages, and the download_parsers needs LANGUAGE_NAMES